exampleconfig.cfg

```diff
@@ -132,6 +153,46 @@ Example config"""
         # text set in [ui]
         # default font: Times 70
 
+    [[actions]]
+        # Actions not listed here won't be available via GUI.
+        # Do note that you may have good reason to e.g. not
+        # provide member searching or library viewing (to get the books,
+        # just search specifying the library)
+        # Script.execute is intentionally omitted here, because
+        # explicit script actions should be used instead.
+        # Apart from that, this list contains all possible actions.
+        edit::Person = Person:edit
+        edit::library = Library:edit
+        edit::group = Group:edit
+        edit::activate_group = Group:activate
+        edit::member = Member:edit
+        edit::change_password = Member:change_password
+        edit::script = Script:edit
+        view::Person = Person:view
+        view::library = Library:view
+        view::group = Group:view
+        view::member = Member:view
+        view::borrow = Borrow:view
+        view::script = Script:view
+        search::Person = Person:search
+        search::library = Library:search
+        search::group = Group:search
+        search::member = Member:search
+        search::borrow = Borrow:search
+        search::script = Script:search
+        book = Book
+        borrow = Borrow:new
+        return = Borrow:restitute
+        leseclub::start = leseclub:start!lua
+        leseclub::end = leseclub:end!lua
+        leseclub::borrow = leseclub:borrow!lua
+        [[[new]]]
+            person = Person:new
+            library = Library:new
+            group = Group:new
+            member = Member:new
+            script = Script:new
+
```

Comment:
Completely move to namespaces. That the ``::`` can be used is actually something between implementation detail and bug.